backend/tasks/tests/test_user.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import Mock, patch
```

Comment:
Do you want to move tests outside dir of the tasks directory? This way we have all the resources under `tasks`. If you want you can create a `backend/tests/tasks/`dir to store the tests for tasks.

Reply:
sure let me update